x-pack/plugins/session_view/public/components/Process/index.tsx

```diff
@@ -268,12 +273,28 @@ function Process({ process, isSessionLeader = false, depth = 0, onProcessSelecte
     onProcessSelected(process);
   }
 
+  /**
+   * calls up to terminal to show details for this command
+   */
+  function onProcessKeyPress(e: KeyboardEvent<HTMLDivElement>): void {
+    e.stopPropagation();
+
+    const selection = window.getSelection();
+
+    // do not select the command if the user was just selecting text for copy.
+    if (selection && selection.type === 'Range') {
+      return;
+    }
+
+    onProcessSelected(process);
+  }
+
   const id = process.getEntityID();
 
   return (
     <>
       <div data-id={id} key={id} css={processCSS}>
-        <div css={wrapperCSS} onClick={onProcessClicked}>
+        <div css={wrapperCSS} onClick={onProcessClicked} onKeyPress={onProcessKeyPress}>
```

Comment:
Why are we adding onKeyPress? Did keyboard navigation get added while I wasn't looking?

Reply:
it's an Eslint rule for Accessibility (all visible clickable elements must also have a keypress handler)

Reply:
won't this get called for all onKeyPress events, or is it only if the element is focused?

Reply:
only the focused, but I just realized that we don't have any mechanism to focus the process using tab key navigation, and also it would need to be filtered for the "Enter" key only, so I just disabled the Eslint rule for now, since it wouldn't work as expected

Reply:
keyboard nav would be cool.

Reply:
yeah, and also a11y compliant, so we can think about adding that